### ISSUE TITLE
Support for Laravel 9

### ISSUE THREAD

**Issue by tmbenhura** [dmcbrn/laravel-email-database-log#12]
Attempting to install gives an error:

Problem 1
- Root composer.json requires dmcbrn/laravel-email-database-log ^5.2 -> satisfiable by dmcbrn/laravel-email-database-log[5.2.0, 5.2.1, 5.2.2].
- dmcbrn/laravel-email-database-log[5.2.0, ..., 5.2.2] require illuminate/support ~8.0 -> found illuminate/support[v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.

**Comment by dmcbrn** [comment]
Sorry I keep missing these.

Easiest to resolve this by forking the library and in `composer.json` replace

```
"require": {
        "illuminate/support": "~8.0",
        "illuminate/database": "~8.0",
        "illuminate/mail": "~8.0",
        "illuminate/queue": "~8.0",
        "doctrine/dbal": "*"
    },
```

with

```
"require": {
        "illuminate/support": "~8.0 || ~9.0",
        "illuminate/database": "~8.0 || ~9.0",
        "illuminate/mail": "~8.0 || ~9.0",
        "illuminate/queue": "~8.0 || ~9.0",
        "doctrine/dbal": "*"
    },
```

There shouldn't be any breaking changes. Let me know if it works or not.

Leaving links for reference:

https://getcomposer.org/doc/articles/versions.md#tilde-version-range-
https://getcomposer.org/doc/articles/versions.md#version-range

I guess the range could be updated to `~5.0 || ~6.0 || ~7.0 || ~8.0 || ~9.0 ||  ~10.0` but I don't have time to test it all out.

**Comment by dmcbrn** [comment]
I just realized it needs to be `~` as minor versions introduce breaking changes.

**Comment by tmbenhura** [comment]
In semantic versioning, minor versions, introduce backwards compatible
changes. This means that any features the package depends on should still
work correctly.

https://semver.org/

**Comment by dmcbrn** [comment]
Yup, I realized I mixed up few things in my head and deleted the comment few minutes later. It was very late in my defense.

There are possible issue with adding L9 support: https://github.com/dmcbrn/laravel-email-database-log/pull/14#issuecomment-1477438156

Leaving open until I have time to test it out in L9.
